templates/_partials/microdata-jsonld.tpl

```diff
@@ -35,6 +35,8 @@
             <meta property="og:image" content="{$p_img.large.url}"/>
         {/foreach}
     {/if}
+{elseif $page.page_name == 'category'}
```

Comment:
```suggestion
{elseif $page.page_name === 'category'}
```

Prefer using strict operators :)

Reply:
Also, check us image exist before, overtwhise you'll end with some og:image metadata with a wrong url, which will lead to a bug on social networks